SampleCode-V5/android-sdk-v5-sample/app-aircraft/src/main/java/dji/sampleV5/aircraft/control/PachKeyManager.kt

```diff
@@ -1025,6 +1097,23 @@ class PachKeyManager() {
             return 90-res
         }
     }
+
+    interface WaypointListener {
```

Comment:
Add documentation.

Reply:
This is something that was left in for the waypoint stuff. I may go with a different implementation, so for now it has just been commented out.